ui/src/app/shared/tile/tile.component.html

```diff
@@ -2,6 +2,6 @@
       <h3>{{title}}</h3>
       <p>{{description}}</p>
 
-      <button id="button" *ngIf="!externalUrl" mat-button (click)="submit()" style="color: white;background-color: {{color}};" class="btn btn-primary">{{buttonName}}</button>
+      <button id="button" *ngIf="!externalUrl" mat-button (click)="submit()" style="color: white; background-color: {{color;}}" class="btn btn-primary">{{buttonName}}</button>
```

Comment:
Should the semi-colon be inside the curly braces for `{{color;}}` or should that still be outside?